### PR TITLE
Batch of runtime fixes.

### LINKED ISSUE #49566: Runtime in special.dm, line 187: Cannot read null.required_reagents
Happened while the server was setting up on locally run code. Reproduced it by spawning **/obj/item/paper/secretrecipe**.
```
[23:15:05] Runtime in special.dm, line 187: Cannot read null.required_reagents
proc name: UpdateInfo (/obj/item/paper/secretrecipe/proc/UpdateInfo)
src: the old recipe (/obj/item/paper/secretrecipe)
src.loc: the notebook floor (168,170,4) (/turf/open/indestructible/paper)
call stack:
the old recipe (/obj/item/paper/secretrecipe): UpdateInfo()
/datum/callback (/datum/callback): InvokeAsync()
Ticker (/datum/controller/subsystem/ticker): setup()
Ticker (/datum/controller/subsystem/ticker): fire()
Ticker (/datum/controller/subsystem/ticker): fire()
Ticker (/datum/controller/subsystem/ticker): fire(0)
Ticker (/datum/controller/subsystem/ticker): ignite(0)
```

### PR DESCRIPTION
**Pull request by AnturK**
Bees nulled base poison type reagent, rest are self-explanatory.

Fixes #49566

### CODE REVIEW

**Comment by Arkatos1** [comment]
Fixes #49566

File: code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 1]
```diff
@@ -159,6 +159,11 @@
 				beegent.reaction_mob(L, INJECT)
 				L.reagents.add_reagent(beegent.type, rand(1,5))
 
+/mob/living/simple_animal/hostile/poison/bees/inject_poison(mob/living/L)
+	if(beegent && istype(L))
```
Comment (spookydonut) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]:
```
if(!beegent || !istype(L) || !L.reagents)
    return
 beegent.reaction_mob(L, INJECT)
L.reagents.add_reagent(beegent.type, rand(1,5))
```

File: code/modules/mob/living/simple_animal/guardian/types/gravitokinetic.dm [hunk 2]
```diff
@@ -49,10 +49,11 @@
 			remove_gravity(i)
 
 /mob/living/simple_animal/hostile/guardian/gravitokinetic/proc/add_gravity(atom/A, new_gravity = 2)
-	A.AddElement(/datum/element/forced_gravity, new_gravity)
-	gravito_targets[A] = new_gravity
-	RegisterSignal(A, COMSIG_MOVABLE_MOVED, .proc/__distance_check)
-	playsound(src, 'sound/effects/gravhit.ogg', 100, TRUE)
+	if(!gravito_targets[A])
```
Comment (spookydonut) [inline code/modules/mob/living/simple_animal/guardian/types/gravitokinetic.dm]: y u not early return

**Comment by nemvar** [comment]
My bad on the secret sauce runtime. Should have tested better.

**Pull request merged**